apps/site/src/lib/components/Navbar/Navbar.tsx

```diff
@@ -66,7 +71,19 @@ function Navbar() {
 					<NavLinkItem href="/">Schedule</NavLinkItem>
 					<NavLinkItem href="/">Resources</NavLinkItem>
 					<NavLinkItem href="/">Stage</NavLinkItem> */}
-					<Button text="Login" href="/login" isLightVersion />
+					{!status && <NavLinkItem href="/apply">Apply</NavLinkItem>}
+					{status !== null && (
+						<NavLinkItem href="/portal">Portal</NavLinkItem>
+					)}
+					{uid === null ? (
```

Comment:
Nitpick [non-blocking]: would prefer naming this as `isLoggedIn` or alike so the intent is clear.